server.go

```diff
@@ -262,11 +270,32 @@ func (s *Server) ReadSocket(packetPool *sync.Pool, reuseport bool) {
 
 // HTTPServe starts the HTTP server and listens perpetually until it encounters an unrecoverable error.
 func (s *Server) HTTPServe() {
+	var prf interface {
+		Stop()
+	}
+
+	once := sync.Once{}
+
+	if s.debug {
+		profileStartOnce.Do(func() {
+			prf = profile.Start()
+		})
+
+		defer func() {
```

Comment:
I think from our testing we determined that it was? I'm inclined to leave it in defensively unless we can prove that it's not possible.